src/components/TabsPage/TabsPage.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { Tab } from '../Tab/Tab';
+
+export const TabsPage = ({ match, tabs }) => {
+  const currentTabId = match.params.id;
+
+  return (
+    <>
+      <ul
+        className="nav nav-pills mb-3 justify-content-center"
+        id="pills-tab"
+        role="tablist"
+      >
+        {tabs.map((tab, i) => (
+          <Tab tab={tab} index={i} />
+        ))}
+      </ul>
+      {tabs.some(elem => elem.id === currentTabId) && (
+        <p className="tab-content">
+          {tabs.find(elem => elem.id === currentTabId).content}
```

Comment:
чтобы упростить читабельность, результаты этих методов можно выносить в константы перед `return`
`const { content } = tabs.find(elem => elem.id === currentTabId);`

а тут просто  
```suggestion
          {content}
```

подобное можно сделать и с `some` методом